app/models/rubygem.rb

```diff
@@ -275,7 +263,17 @@ def protected_days
   end
 
   def reverse_dependencies
-    self.class.reverse_dependencies(name)
+    self.class.joins("inner join versions as v on v.rubygem_id = rubygems.id
+      inner join dependencies as d on d.version_id = v.id").where("v.indexed = 't'
+      and v.position = 0 and d.rubygem_id = ?", id)
```

Comment:
> So now we only revert the dependencies for latest version?

Yes. we show a gem as reverse dependency only if its latest version depends on any of the versions of the dependency. [This assertion](https://github.com/rubygems/rubygems.org/pull/1574/files#diff-ee96e2d5cc5ac47f847e19de030867cdR231) intends to document the case where the latest version of `gem_three` no longer depends on `dependency`.

> Should we change the method name? Or add docs?

I will update [guide](http://guides.rubygems.org/rubygems-org-api/#get---apiv1gemsgem-namereverse_dependenciesjson) when this gets merged :+1: